src/fty_info_server.cc

```diff
@@ -121,21 +121,26 @@ info_server_destroy (fty_info_server_t  **self_p)
 }
 
 
-//return IPC (uuid first 8 digits)
+//return NAME (uuid first 8 digits)
 // the returned buffer should be freed
 static
-char *s_get_name(const char *name, const char *uuid)
+char *s_get_name(ftyinfo_t *info)
 {
-
-    char *buffer = (char*)malloc(strlen(name)+12);
+    std::string s_name = SRV_IPC_NAME;
```

Comment:
Constant name  SRV_IPC_NAME to SRV_DEFAULT_NAME ?

Reply:
logged a reminder issue for now, thx for your comment